spec/models/user_spec.rb

```diff
@@ -2,4 +2,16 @@
 
 RSpec.describe User, type: :model do
   pending "add some examples to (or delete) #{__FILE__}"
+
```

Comment:
- [ ] [OPTIONAL] This is optional because the previous reviewer did not point it out. Please remove all pending or not-implemented tests such as this one to make your test files cleaner